schema.graphql

```diff
@@ -2,24 +2,158 @@ type Hat @entity {
   id: ID!
```

Comment:
I love the pretty id! Though I think its probably best used as a reference (additional attribute) rather than the main id for a hat.